src/pages/components/HistoryItem.qml

```diff
@@ -0,0 +1,106 @@
+import QtQuick 2.6
+import Sailfish.Silica 1.0
+import Sailfish.Browser 1.0
+
+ListItem {
+    id: root
+
+    property string search
+
+    function remove() {
+        view.model.remove(model.index)
+    }
+
+    function openLink() {
+        webView.load(model.url, model.title)
+        pageStack.pop()
+    }
+
+    width: view.width
+    menu: contextMenuComponent
+
+    ListView.onAdd: AddAnimation { target: root }
+
```

Comment:
Icon missing.

Icon should have... Horizontal page margin at the left and Theme.paddingMedium between icon and those two rows.

Reply:
At the moment I don't know how to store icons for history. Maybe add it in next PR..